app/design/adminhtml/Magento/backend/web/css/styles-old.less

```diff
@@ -1076,29 +1076,6 @@
         margin-top: 4px;
     }
 
-    .control {
```

Comment:
I have a concern regarding backward compatibility aspects of removing these styles. Extensions may relay on these styles. I'd try to find a fix without removing this less block